examples-standalone/grid-charts-integration/src/app/app.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-root',
+    styleUrls: ['./app.component.scss'],

```

Comment:
Remove empty app.component.scss file, there are no styles. Also, remove this line.